### PR TITLE
[NCL-6906] Add AlignmentPreference.

### PR DESCRIPTION
**Pull request by matejonnet**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Review by janinko — CHANGES_REQUESTED** [review]

File: rest-api/pom.xml [hunk 1]
```diff
@@ -38,10 +38,19 @@
...
+            <artifactId>pnc-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.jboss.pnc</groupId>
             <artifactId>dto</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <scope>provided</scope>
```
Comment (janinko) [inline rest-api/pom.xml:52]: It's used in tests only? If so, please change it to test scope

**Review by janinko — CHANGES_REQUESTED** [review]

File: rest-api/pom.xml (hunk 1 above)
Comment (matejonnet) [reply rest-api/pom.xml:52]: It's the same as we have in the DTO module.

Comment (janinko) [reply rest-api/pom.xml:52]: but in DTO it¨s because of org.hibernate.validator.constraints.URL that is used in DeliverablesAnalysisRequest

Comment (matejonnet) [reply rest-api/pom.xml:52]: ok, changing it to test.

**Review by janinko — APPROVED** [review]

**Pull request merged**